test/ActiveLogin.Authentication.BankId.AspNetCore.Test/Helpers/TokenHelper.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Text.RegularExpressions;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Test.Helpers
+{
+    public static class TokenHelper
+    {
+        public static string ExtractToken(string responseText)
```

Comment:
Suggestion to rename it to `ExtractRequestVerificationTokenFromForm`.

Reply:
Absolutely!